test/functional/apps/visualize/_tile_map.js

```diff
@@ -88,9 +88,6 @@ export default function ({ getService, getPageObjects }) {
           //level 0
           return PageObjects.visualize.clickMapZoomOut();
         })
-        .then(function sleep() {
-          return PageObjects.common.sleep(2000);
-        })
```

Comment:
removed this. snuck in during troubleshooting but not necessary strictly speaking.